crates/mfem-sys/build.rs

```diff
@@ -32,7 +32,7 @@ fn main() {
         .flag_if_supported("-O3")
         .flag_if_supported("-w");
     for f in mfem_config.cxx_flags {
-        build.flag_if_supported(f);
+        build.flag_if_supported(&f);
```

Comment:
This seems to be necessary for non-nightly Rust.

Reply:
Given that it compiles on the CI for stable Rust, I would guess it is rather related to (bumping) the version of a package.